src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts

```diff
@@ -1586,10 +1586,12 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
 		this._lastCellWithEditorFocus = cell;
 	}
 
-	private async _warmupWithMarkdownRenderer(viewModel: NotebookViewModel, viewState: INotebookEditorViewState | undefined) {
+	private async _warmupWithMarkdownRenderer(viewModel: NotebookViewModel, viewState: INotebookEditorViewState | undefined, perf?: NotebookPerfMarks) {
 
 		this.logService.debug('NotebookEditorWidget', 'warmup ' + this.viewModel?.uri.toString());
 		await this._resolveWebview();
+		perf?.mark('webviewCommLoaded');
```

Comment:
@amunger I fixed this webview perf mark so we can differentiate between webview initialization and the markup cells viewport warmup to help us understand which one is slowing down the file opening.